zshrc.sh

```diff
@@ -97,5 +101,6 @@ ZSH_THEME_GIT_PROMPT_BEHIND="%{↓%G%}"
 ZSH_THEME_GIT_PROMPT_AHEAD="%{↑%G%}"
 ZSH_THEME_GIT_PROMPT_UNTRACKED="%{…%G%}"
 ZSH_THEME_GIT_PROMPT_CLEAN="%{$fg_bold[green]%}%{✔%G%}"
+ZSH_THEME_GIT_PROMPT_STASHES="%{$fg_bold[yellow]%}%{⚑%G%}"
```

Comment:
Only the branch and the checkmark are bold. Status numbers aren't. Maybe this should just be $fg[yellow]?